compass/landice/tests/ismip6_run_ais/ismip6_run_ais.cfg

```diff
@@ -0,0 +1,44 @@
+# config options for ismip6 antarctic ice sheet data set
+[ismip6_run_ais]
+
+# Resolution that will be set up.  Should be one of 08, 04 with leading 0.  The paths below must manually be set to be consistent
+mesh_res = 08
```

Comment:
Why with a leading zero?  Probably state that it is the highest resolution in km.

Reply:
```suggestion
# Finest resolution of the MALI mesh that will be used in km (currently either 4 or 8). The paths below must
# be set for files on a consistent mesh
mesh_res = 8
```

Reply:
Because the ISMIP6 submission categorizes mesh resolution as one of 16, 08, 04, 02, 01, we've adopted that naming convention to refer to our meshes.  When the results get uploaded they require a directory name with the resolution including the leading zero, so I've adopted it everywhere here.

Reply:
I've gone ahead and taken your suggestion to have it be an integer in the cfg file and padded with zeros as needed.